docs/src/index.md

```diff
@@ -3,11 +3,16 @@
 Welcome to the DataFrames.jl documentation!
 
 This resource aims to teach you everything you need to know to get up and
-running with tabular data manipulation using the DataFrames.jl package. For more
-illustrations of its usage in conjunction with other packages, the [DataFrames
-Tutorial using Jupyter
-Notebooks](https://github.com/bkamins/Julia-DataFrames-Tutorial/) is a good
-complementary resource.
+running with tabular data manipulation using the DataFrames.jl package.
+
+For more illustrations of DataFrames.jl usage, in particular in conjunction with
+other packages you can check-out the following resources
+(they are kept up to date with the released version of DataFrames.jl):
+* [Data Wrangling with DataFrames.jl Cheat Sheet](https://www.ahsmart.com/pub/data-wrangling-with-data-frames-jl-cheat-sheet/)
+* [DataFrames Tutorial using Jupyter Notebooks](https://github.com/bkamins/Julia-DataFrames-Tutorial/)
+* [Julia Academy DataFrames.jl tutorial](https://github.com/JuliaAcademy/DataFrames)
+* [JuliaCon 2019](https://github.com/bkamins/JuliaCon2019-DataFrames-Tutorial), [JuliaCon 2020](https://github.com/bkamins/JuliaCon2020-DataFrames-Tutorial), and [PyData Global 2020](https://github.com/bkamins/PyDataGlobal2020) tutorials
+* [Showcasing DataFrames.jl](https://gist.github.com/bkamins/c6b7b536aefda6c2be67f133972a62e8) gist
```

Comment:
Why don't you give a nicer URL for that page? :-D

Reply:
OK - I can move it to a GitHub repo. Would you prefer it?

Reply:
As you like, but if you intend to update it in the long term that would make sense IMHO.

Reply:
I want to maintain it (that is why I put it in this list not to forget :smile: )